packages/spark-core/utilities/_utilities.scss

```diff
@@ -2,166 +2,301 @@
 // Utilities
 // ==================================================================
 
-// TODO: Dave is doing utils
-.sprk-u-Hide {
-  display: none;
+//
+// Single property utilities
+//
+// These utilities should map directly to a single CSS property and
+// value and should always use !important.
+//
+// Properties with vendor prefixes count as single properties.
+///
+
+// Display and Visibility
+.sprk-u-Display--none {
+  display: none !important;
```

Comment:
When this PR comes to a close (or maybe it should be a part of this PR) - I recommend going through and finding all the `display: none` places that are currently in the code and removing them and adjusting to use these.